test/library/packages/LinearAlgebra/performance/dot-perf.perfcompopts

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+system=$($CHPL_HOME/util/printchplenv | grep 'CHPL_TARGET_PLATFORM' | cut -d\  -f2)
+
+if [ "$system" = "cray-cs" ] || [ "$system" = "cray-xc" ]
```

Comment:
We should drop cs here because we need to adjust our config to load libsci in nightly testing. Moreover, as it is a single locale performance test with its current shape, this'll be only perf tested on a linux64 system.

@ben-albrecht -- Should we still have the rest for XC, and create a distributed version of this test in the future, or do we want it to be distributed now?

Reply:
Should I add ml config files with 1 locale?

Reply:
>  Should we still have the rest for XC, and create a distributed version of this test in the future, or do we want it to be distributed now?

Let's do it later.

> Should I add ml config files with 1 locale?

I don't think that's necessary.